src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -65,6 +68,12 @@ def _ensure_monitor_with_config(
     except Monitor.DoesNotExist:
         monitor = None
 
+    # It is unusual if a monitor already exists but was not already assigned a
+    # seat in the quotas system. Capture an error for this situation
+    if monitor and quotas_outcome == PermitCheckInStatus.ACCEPTED_FOR_UPSERT:
```

Comment:
@mbauer404 should I do this here actually, or should we log this in the quotas `assign_monitor_seat` in getsentry instead?

Reply:
yeah might be better to do it in getsentry